CONTRIBUTING.md

```diff
@@ -674,7 +674,7 @@ ccache -M 25Gi
 ```
 
 To check this is working, do two clean builds of pytorch in a row. The second
-build should be substantially and noticeably faster than the first build.
+build should be substantially and noticeably faster than the first build. If this doesn't seem to be the case, check that each of the symlinks above actually link to your `ccache`. For example, running `readlink -e $(which g++)` on your Linux terminal should return `~/ccache/bin/ccache` if you installed `ccache` from source.
```

Comment:
Suggest leave out the build from source part. what you described here is how to check if ccache was used correctly
```suggestion
build should be substantially and noticeably faster than the first build. If this doesn't seem to be the case, check that each of the symlinks above actually link to your `ccache`. For example, running `readlink -e $(which g++)` on your Linux terminal should return `~/ccache/bin/ccache` if you installed `ccache` correctly.
```

Reply:
@walterddr given the discussion above, I don't think that's quite accurate

Reply:
Thanks! I included that specifically because if people installed ccache through other means, the path would be different, right? (Like Sam's comment above with ccache from conda.) Would that be misleading to remove the "from source"?

Reply:
which part is not accurate? i think checking the symlinks actually links to `ccache` is useful regardless of whether one install from source or from conda. correct? 

Or are you saying the `~/ccache/bin/ccache` part is misleading? (since this is not the conda instalation path)

Reply:
@walterddr right, I meant the latter; that's why I'm suggesting that this PR actually show what the path would look like for each of the three different installation methods

Reply:
AHH... I see. that make sense. in this case can we change it to "the symlink should link to your installation of ccache (or having ccache in the path?)

Reply:
@walterddr you mean as opposed to actually listing specific paths? I think that would be worse, since the point of this PR is to help readers be able to easily check _which_ method they used to install ccache

Reply:
or actually maybe I misunderstood the point of the PR 😅 @janeyx99?

Reply:
Ah, sorry, the point was to have people check their symlinks and make sure it linked to wherever they installed ccache. What about the below?

```suggestion
build should be substantially and noticeably faster than the first build. If this doesn't seem to be the case, check that each of the symlinks above actually link to your installation of `ccache`. For example, if you followed the first option and installed `ccache` from source on a Linux machine, running `readlink -e $(which g++)` should return `~/ccache/bin/ccache`.
```

Reply:
Yeah that works for me :+1: sorry for the noise on this PR discussion